lib/spotifuby/spotify/web.rb

```diff
@@ -35,7 +35,12 @@ def who_added_track(user, playlist_uri, track_uri)
           all_songs.merge!(playlist.tracks_added_by)
         end
 
-        { name: all_songs[track_id].id }
+        listing = all_songs[track_id]
+        # default answer when current song is not on the playlist
+        return { name: 'Disco Stu' } if (listing.nil? || listing.empty?)
```

Comment:
My only issue with this is the Disco Stu part. I'd like to centralize that with the bot so it's all in one place (client.rb for ruby, hubot script for js). Could you make it return nil instead and then return Disco Stu there?